packages/lit-dev-content/site/tutorials/content/working-with-lists/01.md

```diff
@@ -0,0 +1,18 @@
+If we're not working with arrays, we can also create our own array of
```

Comment:
Hmm this set not being an array doesn't seem to be the problem in this step that you're tackling. Lit can still render a Set because it's an iterable e.g. just render `this.items` and see.

The problem being solved here is that you need to run .map on an iterable but .map is only available on Array not the generic iterable. Thus you can use `for...of`.

I'd also consider changing the title of this step to address the crux of lesson like `Using for Loops`